### DISCUSSION TITLE
force-onlink路由支持

### DISCUSSION THREAD

**Discussion by xmxs** [bohanwood/debi#83] [Ideas]
![force_onlink_route](https://user-images.githubusercontent.com/5410657/212617632-4715015d-5700-48f0-99ba-0e43e6b7dba3.png)
![network_config](https://user-images.githubusercontent.com/5410657/212617882-135062d3-78ae-4328-a550-32951060a978.png)
是否可以考虑添加对force-onlink路由的支持呢，当前这种情况下，会因为IP和网关不在同一个网段而执行失败。

**Comment by bohanwood** [comment]
我一直想处理这种情况的，但是不知道 Debian preseed 里面怎么设置。
